src/cs/lib/msquic_generated.cs

```diff
@@ -44,6 +44,25 @@ internal enum QUIC_LOAD_BALANCING_MODE
         COUNT,
     }
 
+    internal enum QUIC_TLS_ALERT_CODES
+    {
+        SUCCESS = 0xFFFF,
```

Comment:
This is larger than the MAX however?

Reply:
Yeah, on purpose because while it's a valid enum, it's not a valid wire-format TLS alert.